js_formatter.py

```diff
@@ -34,20 +32,13 @@ def _try_get_short_path(path):
 # crazyness to get jsbeautifier.unpackers to actually import
 # with sublime's weird hackery of the path and module loading
 add_lib_path(libs_path)
-add_lib_path(os.path.join(libs_path, "six-1.8.0"))
-add_lib_path(os.path.join(libs_path, "jsbeautifier"))
-add_lib_path(os.path.join(libs_path, "jsbeautifier", "jsbeautifier"))
```

Comment:
six was not a module, I've renamed it : libs/{six-1.8.0/six.py → six/__init__.py}
jsbeautify was not a module : libs/jsbeautifier/{jsbeautifier → }/__init__.py